socorro/signature/rules.py

```diff
@@ -226,15 +201,7 @@ def normalize_frame(
         names of the fields from the jsonMDSW frame output. This allows this
         function to be invoked by passing a frame as ``**a_frame``.
 
-        Sometimes, a frame may already have a normalized version cached. If
-        that exists, return it instead.
```

Comment:
Because we change the normalization code periodically, it's not helpful to cache the normalized form. Because of that, I wrote code elsewhere to remove the normalized data. So... logically if we're just going to remove it and go out of our way not to use it, we shouldn't be keeping it in the first place. Thus I removed caching of the normalized form.